client/src/pages/ManageJobPage/BlankComponent.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: 360,
+    height: 125,
+    marginTop: theme.spacing(1),
+  },
+  contents: {
+    align: 'center',
+    marginTop: theme.spacing(4),
+  },
+}));
+
+export default function BlankComponent(props) {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.root} variant="outlined">
+      <CardContent className={classes.contents}>
+        <Typography variant="h6" gutterBottom>
+          No jobs..
```

Comment:
This should say something like "No jobs right now." And maybe something like "Make sure your profile is updated and your availability is set here!" With a link to the edit profile page

Reply:
Thank you for good ideas!